src/resolvers-cassandra/Settings/index.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
 
   siteTerms: queries.siteTerms,
   sites: queries.sites,
+  trustedsources: queries.trustedsources,
```

Comment:
No camel case?

Reply:
good point. I'll change it to Camel Case for consistency.